css-media-queries/breakpoints.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Media Queries - Breakpoints</title>
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="pokemon.css">
+  <style>
+
+    #card-deck {
+      display: flex;
+      flex-wrap: wrap;
+    }
+
+    .card-wrapper {
+      padding: 0.5rem;
+      flex-basis: 100%;
+    }
+
+    /* your code here 👇 */
+    /* @media (min-width: 575px) {
+      .card-wrapper {
+        flex-basis: 50%;
+      }
+    }
+
+    @media (min-width: 767px) {
+      .card-wrapper {
+        flex-basis: calc(100%/3)
+      }
+    } */
```

Comment:
Any particular reason these are commented out?

Reply:
Oh!!  I commented them out to see the difference before and after. I forgot to uncomment them out haha